app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,13 +64,17 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [])
+    public function transform($data, $code = 200, $meta = [], $include = null)
```

Comment:
Yeah, that's a good thought too. I think the only place we regularly use the `$meta` argument is in the `paginatedCollection` method, which is safely tucked away in the base controller.